webhook/webhook.go

```diff
@@ -281,6 +283,19 @@ func (ac *AdmissionController) Run(stop <-chan struct{}) error {
 		TLSConfig: tlsConfig,
 	}
 
+	serverBootstrapErrCh := make(chan struct{})
+	wg.Add(1)
+	go func() {
+		if err := server.ListenAndServeTLS("", ""); err != nil {
+			logger.Errorw("ListenAndServeTLS for admission webhook returned error", zap.Error(err))
+			close(serverBootstrapErrCh)
+		}
+		wg.Done()
+		logger.Info("server ready to listen")
+	}()
+
+	wg.Wait()
```

Comment:
This is equivalent to not doing this in a go-routine, which I believe is incorrect given the examples:
https://golang.org/pkg/net/http/#ListenAndServe

Reply:
+1 the way it's written this `wg` won't be ready until after the server starts. You need one more goroutine to probe the server, I guess, and in that goroutine trigger the `wg.Done`.

Reply:
Thanks for your review. @mattmoor @vagababov 
Now I try to find a method to make sure the server is ready to create. Can I use the `http.Get()` method to access the server to check the server is ready?